shub/deploy_reqs.py

```diff
@@ -28,15 +28,15 @@
 
 @click.command(help=HELP, short_help=SHORT_HELP)
 @click.argument("target", required=False, default="default")
-@click.option("-r", "--requirements-file", default='requirements.txt',
+@click.option("-r", "--requirements-file", default=None,
               type=click.STRING)
 def cli(target, requirements_file):
     main(target, requirements_file)
 
 
 def main(target, requirements_file):
     targetconf = get_target_conf(target)
-    requirements_full_path = os.path.abspath(requirements_file)
+    requirements_full_path = os.path.abspath(requirements_file or targetconf.requirement_files)
```

Comment:
In case we decide to merge this despite https://github.com/scrapinghub/shub/pull/190#issuecomment-224552033: This should be `targetconf.requirements_file`.